frontend/packages/dev-console/src/utils/pipeline-actions.tsx

```diff
@@ -131,13 +131,32 @@ export const startPipeline: KebabAction = (
   },
 });
 
-export const rerunPipeline: KebabAction = (kind: K8sKind, pipelineRun: PipelineRun) => {
-  // The returned function will be called using the 'kind' and 'obj' in Kebab Actions
+type RerunPipelineData = {
+  onComplete?: (pipelineRun: PipelineRun) => void;
+};
+const rerunPipeline: KebabAction = (
```

Comment:
I logged https://jira.coreos.com/browse/ODC-2184 to address the fact that this function was converted to being a rereun pipeline runs.